### PR TITLE
[OPENTOK-44970] add ws headers

### PR DESCRIPTION
**Pull request by jvm7293**

### CODE REVIEW

**Review by phil-lavin — APPROVED** [review]

File: mod_wsbridge/mod_wsbridge.c [hunk 1]
```diff
@@ -841,6 +859,17 @@ static switch_status_t channel_on_init(switch_core_session_t *session)
...
 
+	if (cJSON_GetArraySize(tech_pvt->message) == 1 && cJSON_GetObjectItem(tech_pvt->message, "content-type")) {
+		cJSON* json_req = NULL;
+		if ((json_req = get_ws_headers(channel))) {
+			cJSON_Delete(tech_pvt->message);
+			tech_pvt->message = json_req;
+			cJSON_AddItemToObject(json_req, "content-type", cJSON_CreateString(tech_pvt->content_type));
+		}
+	}
+	switch_log_printf(SWITCH_CHANNEL_LOG, SWITCH_LOG_NOTICE, "%s: tech_pvt->message = [%s]\n",
```
Comment (brunorib) [inline mod_wsbridge/mod_wsbridge.c]: Just a nit, maybe we could log this inside the if (If message has been changed). When I executed it I got two logs and was unsure if it was doing everything again two times, it seems not but logs twice.

**Review by brunorib — APPROVED** [review]
LGTM

File: mod_wsbridge/mod_wsbridge.c (hunk 1 above)
Comment (jvm7293) [reply mod_wsbridge/mod_wsbridge.c]: Done.

**Review by salman-vonage — APPROVED** [review]

**Review by odivorra — APPROVED** [review]
LGTM

**Pull request merged**